README.md

```diff
@@ -1,4 +1,63 @@
 
 ## Overview
 
-The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
\ No newline at end of file
+The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
+
+Inspired by the [Kubernetes enhancement]{https://github.com/kubernetes/enhancements} process.
+
+This repository provides a rally point to discuss, debate, and reach consensus for how Open Cluster Management enhancements are introduced.
+
+Enhancements may take multiple releases to ultimately complete and thus provide the basis of a community roadmap. Enhancements may be filed from anyone in the community, but require consensus from domain specific project maintainers in order to implement and accept into the release.
+
+For template references, see the [template]{https://github.com/open-cluster-management/enhancements/blob/master/guidelines/enhancement_template.md}.
+
+## Is My Thing an Enhancement?
+
+A rough heuristic for an enhancement is anything that:
+
+- impacts how multiple clusters are managed including addition or removal of significant
+  capabilities
+- impacts upgrade/downgrade 
+- needs significant effort to complete
+- requires consensus/code across multiple domains/repositories
+- has phases of maturity (Dev Preview, Tech Preview, GA)
+- demands formal documentation to utilize
+
+It is unlikely to require an enhancement if it:
+
+- fixes a bug
+- adds more testing
+- internally refactors a code or component only visible to that components
+  domain
+- minimal impact to distribution as a whole
+
+If you are not sure if the proposed work requires an enhancement, file an issue
+and ask!
+
+## When to Create a New Enhancement
+
+Create an enhancement here once you:
+
+- have circulated your idea to see if there is interest
+- (optionally) have done a prototype in your own fork
+- have identified people who agree to work on and maintain the enhancement
+  - many enhancements will take several releases to complete  
+
+## Why are Enhancements Tracked
+
+As the project evolves, its important that the open-cluster-management community understands how we
+build, test, and document our work.  Individually it is hard to understand how
+all parts of the system interact, but as a community we can lean on each other
+to build the right design and approach before getting too deep into an
+implementation.
+
+## When to Comment on an Enhancement Issue
+
+Please comment on the enhancement issue to:
+- request a review or clarification on the process
+- update status of the enhancement effort
+- link to relevant issues in other repos
+
+Please do not comment on the enhancement issue to:
```

Comment:
Rejections can be particularly sensitive.  In kube we had cases of enhancements being open for long periods of time before someone was willing to make a statement.

semi-related - we have trouble in kube with enhancements getting stuck in alpha or beta state and not having a good way to finish them either way.  You may want to think about it.  No need to go all the way to something like https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/1635-prevent-permabeta/README.md, but it's a thing that happens.

Reply:
I would like the enhancement review process to flow through what is now the Architecture SIG to discuss viability and through the Steering Committee to discuss business value/alignment with mission charter. 

I would incorporate the API graduation/deprecation policy captured as part of the enhancements process (e.g. everyone should be thinking about the lifecycle of their API upfront and how long it may need to be maintained; what would cause it to become less relevant/merit deprecation.)